Server/JokesOnYou.Web.Api/DTOs/JokeDto.cs

```diff
@@ -2,12 +2,12 @@
 
 namespace JokesOnYou.Web.Api.DTOs
 {
-    public class JokeReplyDto
+    public class JokeDto
     {
         public int Id { get; set; }
         public string Premise { get; set; }
         public string Punchline { get; set; }
-        public string AuthorId { get; set; }
```

Comment:
I think we just need author id and author username so we could just have them as inline properties

Reply:
We could but its not part of the Joke so I think it should have its own Dto.
And Author Dto could be used for more than just the joke and we might add more then just the UserName.